### PR TITLE
Message component

### PR DESCRIPTION
**Pull request by Jap9**
Pulled latest Master, rebased my branch onto it.
User now have a badge on the "Messages" bell to show unread messages (still not working)
Users can mark their received messages as "Read"

### CODE REVIEW

**Comment by zurfyx** [comment]
Your pull request contains triplicated commits, please read over the following :

Brief explanation of why it occurred after rebasing:
http://stackoverflow.com/q/9264314/2034015

How to get it solved:
http://stackoverflow.com/a/38457832/2034015

**Comment by Jap9** [comment]
@zurfyx 
I always do fetch -> pull Master and then chekout my branch -> Rebase onto Master
I haven't done the fetch this time and was unable to Make the Request, then I did the process again and after doing the process it got duplicated (again) and messed up :(

**Comment by zurfyx** [comment]
I corrected them for you (inc. the lint tests which were not passing). Please, next time you are working on your own branch force push instead, otherwise you will get duplicates once again. If you are working with more people, then do merge instead.

If you plan to continue working on this branch, the best thing you can do is to delete your local copy and check it out as a new local branch.

File: src/app/navbar/navbar.component.html [hunk 1]
```diff
@@ -14,31 +14,31 @@
         <!--<li><a routerLink="/advertisements" [routerLinkActive]="'active'" [routerLinkActiveOptions]="{ exact: true }">Advertisements</a></li>-->
         <!--<li><a routerLink="/searchAdvertisements" [routerLinkActive]="'active'" [routerLinkActiveOptions]="{ exact: true }"> Search Advertisements</a></li>-->
         <!-- <li><a href="#contact">Contact</a></li> // move to footer -->
-        <app-search-ad></app-search-ad>
+        <!--<app-search-ad></app-search-ad>-->
```
Comment (SrBlimp) [inline src/app/navbar/navbar.component.html]: Thank you for deleting my search. :rage:

if you don't mind It's better not to touch the other part in order to solve your problems. :tractor:

Comment (zurfyx) [reply src/app/navbar/navbar.component.html]: @SrBlimp For some reason search component on navbar was commented, even though it was working correctly. It is now uncommented.

**Comment by rogargon** [comment]
Is the problem with the navbar already solved? Ready for merging?

**Comment by zurfyx** [comment]
@rogargon I rebased it manually onto master since there were conflicts with the navbar and the last pull request you just accepted #49. It should be ready to merge now.

**Pull request merged**